sql-gremlin/src/test/java/org/twilmes/sql/gremlin/adapter/GremlinSqlAggregateTest.java

```diff
@@ -68,4 +68,10 @@ public void testAggregateColumnTypeMixedAgg() throws SQLException {
         // validate the metadata type matches return type
         runQueryTestColumnType("select age, count(age) from person group by age");
     }
+
+    @Test
+    public void testCountStar() throws SQLException {
```

Comment:
Can we add a unit test (not integration test) to compare that translation to gremlin is correct?

Reply:
Same comment as other PR. We could do it but it would ultimately be really hacky. The other concern is that since we are still in development, if we update how our traversal is formed, we would break our tests.

I was thinking of overhauling the current structure - currently it outputs a Map-like structure, I was thinking of migrating to a List-like output for performance reasons, but was going to wait and see first